enigma-swing/src/main/java/cuchaz/enigma/gui/Main.java

```diff
@@ -60,6 +60,8 @@ public static void main(String[] args) throws IOException {
 		parser.acceptsAll(List.of("edit-javadocs", "d"), "Enable editing Javadocs");
 		parser.acceptsAll(List.of("no-edit-javadocs", "D"), "Disable editing Javadocs");
 
+		parser.accepts("single-class-tree", "Shows all classes in the de-obfuscated panel");
```

Comment:
I'd rather say "List obfuscated and deobfuscated classes in a single panel" or "Unify the deobfuscated and obfuscated class panels" or something along those lines, it's not really the deobfuscated panel anymore if it contains everything (except in the code, but shhh don't tell em!) and there's no obfuscated panel to complement it